bundles/org.openhab.ui.start/pages/404.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <!-- Simple HttpErrorPages | MIT License | https://github.com/AndiDittrich/HttpErrorPages -->
```

Comment:
Should this be mentioned in the about.html?

Reply:
added.